experiments/helpers.py

```diff
@@ -0,0 +1,73 @@
+from evaluation.evaluation_metrics import EvaluationMetrics
+from models.unet_model import unet_2d
+from keras.losses import categorical_crossentropy
+from tensorflow.keras.utils import to_categorical
+from tensorflow.keras.callbacks import EarlyStopping
+import pickle
+
+def normalizing(X, y):
+  print('y shape: ', y.shape)
+  print('Normalizing data...')
+  y_one_hot = to_categorical(y, num_classes=3)
+  print('y one hot shape: ', y_one_hot.shape)
+  X_normal = X/255
+  return X_normal, y_one_hot
+
+
+
+def define_model(input_shape=(256, 256, 5), num_classes=3):
+    model = unet_2d(input_shape=input_shape, num_classes=num_classes)
+    model.compile(optimizer='adam',
+                  loss=categorical_crossentropy,
+                  metrics=['accuracy'])
+    return model
+
+def train_model(model, x_train, y_train, x_val, y_val, epochs=100):
+    early_stop = EarlyStopping(monitor='accuracy', patience=5)
+    history = model.fit(x=x_train, y=y_train, epochs=epochs,
+                        validation_data=(x_val, y_val),
+                        callbacks=[early_stop])
+    return history
+
+def save_model(model, model_name, saving_path='../models'):
+    model.save(f'{saving_path}/model_{model_name}.h5')
```

Comment:
I think adding a function and creating two lines is a bit too much overhead for a one liner